eva_cttv_pipeline/utilities.py

```diff
@@ -158,6 +179,7 @@ def __init__(self, argv):
         parser.add_argument("-v", "--variantSummaryFile", dest="variant_summary_file",
                             help="Path to file with RS id to ensembl gene ID and consequence "
                                  "mappings", required=True)
+        parser.add_argument("-j", dest="json_file", help="File with json string per line")
```

Comment:
What kind of information does this contain? Try to write a message thinking that you know nothing about how the system works internally.